var/spack/repos/builtin/packages/tau/package.py

```diff
@@ -49,21 +53,76 @@ class Tau(Package):
     variant('download', default=False,
             description='Downloads and builds various dependencies')
     variant('scorep', default=False, description='Activates SCOREP support')
-    variant('openmp', default=True, description='Use OpenMP threads')
-    variant('mpi', default=True,
+    variant(
+        'otf',
+        default=False,
+        description='Activates support of Open Trace Format (OTF)')
```

Comment:
variants don't have to be wrapped like this -- you can keep them on one line.  They're excepted from the 80-char limit checks.